### ISSUE TITLE
Accessibility: dropdown button lacks discernible text

### ISSUE THREAD

**Issue by folkwest** [OSC/ondemand#5151]
The dropdown button (shown below) does not have an accessible name, so screen readers can’t determine its purpose. This was flagged during an accessibility audit under “Buttons must have discernible text.”

<img width="687" height="346" alt="Image" src="https://github.com/user-attachments/assets/655bdb69-8815-4d4f-97f9-4133c965fef2" />

This issue is related to WCAG 2.1 Success Criterion 4.1.2 - Name, Role, Value, which requires that interactive elements have an accessible name that can be interpreted by assistive technologies.

Element Location:
.actions-btn
```
<button type="button" 
class="actions-btn btn btn-outline-dark btn-sm dropdown-toggle" data-bs-toggle="dropdown" 
aria-haspopup="true" 
aria-expanded="false">
<span class="fa fa-ellipsis-v"></span>
</button>
```

Example Fix (adding aria-label)
```
<button type="button"
        class="actions-btn btn btn-outline-dark btn-sm dropdown-toggle"
        data-bs-toggle="dropdown"
        aria-haspopup="true"
        aria-expanded="false"
        aria-label="Actions">
  <span class="fa fa-ellipsis-v"></span>
</button>
```

**Comment by johrstrom** [comment]
Thanks! we should be able to get to this in 4.2. If you come across anything else, please let us know!
